modules/redpanda/go.mod

```
@@ -1,13 +1,14 @@
 module github.com/testcontainers/testcontainers-go/modules/redpanda
 
-go 1.20
+go 1.21
```

Let's stick with 1.21 unless there is code that needs 1.21. If the latter, then we should update the CI pipeline and the docs warning about the toolchain version (we already do that for the compose module)
```suggestion
go 1.20
```

---

done